metricbeat/templates/daemonset.yaml

```diff
@@ -136,6 +130,13 @@ spec:
           mountPath: /usr/share/metricbeat/{{ $path }}
           readOnly: true
           subPath: {{ $path }}
+        {{ else }}
+        {{- range $path, $config := .Values.daemonset.metricbeatConfig }}
+        - name: metricbeat-config
+          mountPath: /usr/share/metricbeat/{{ $path }}
+          readOnly: true
+          subPath: {{ $path }}
+        {{- end }}
```

Comment:
```suggestion
```

Reply:
This comment was part of https://github.com/elastic/helm-charts/pull/572#discussion_r410199664 and can be ignored.